ocfweb/docs/templates/docs/lab.html

```diff
@@ -131,4 +134,19 @@ <h4 id="walkthrough">Having trouble finding the entrance?</h4>
             </video>
         </div>
     </div>
+
+    <script>
+	function toggle_weekly_hours() {
+		var change = document.getElementById("weekly_hours_btn");
+		if (change.innerHTML == "Show")
+		{
+			change.innerHTML = "Hide";
+			$("#weekly_hours").show();
+		}
+		else {
+			change.innerHTML = "Show";
+			$("#weekly_hours").hide();
+		}
+	}
+    </script>
```

Comment:
Can we use something from Bootstrap instead of reinventing it ourselves?

https://getbootstrap.com/docs/3.3/javascript/#collapse-example looks like basically what you want. Plus we get nice transitions this way.

Reply:
(to clarify, we already have all the bootstrap css/js, so it should be as simple as adding the right classes and hooking things up; the example there should help)